sdk/template/azure-template-stress/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v2
+name: java-template
+description: An example stress test chart for performing azure resource deployments
+version: 0.1.1
+appVersion: v0.1
+annotations:
+  stressTest: 'true'  # enable auto-discovery of this test via `find-all-stress-packages.ps1`
```

Comment:
@benbp, is the final-all-stress-packages.ps1 going to find _this_ template package as well? Or does it auto-ignore some folders?

Reply:
oh, let me change it to false